cypress/integration/login.spec.ts

```diff
@@ -19,16 +19,33 @@ describe('Login process', () => {
         cy.get('[data-test="user-avatar"]').should('be.visible');
     });
 
-    it('clicking on user avatar takes you to My Page', () => {
+    it('takes you to My Page when clicking on user avatar', () => {
         cy.visit('/login');
         cy.get('input[aria-label="E-mail address"]').type('testadmin@example.com');
         cy.get('input[aria-label="Password"]').type('password');
         cy.get('input[aria-label="Log in"]').click();
-        cy.get('[data-test="username"').click();
+        cy.get('[data-test="username"]').click();
         cy.url().should('match', /\/my$/);
     });
 
+    it('contains a logout button wich logs you out and takes you back to the login page', () => {
```

Comment:
It doesn't take you back to the *login* page though, but the *home page*. I think this test description should be updated.